packages/shared/src/utils/storage/storage_events.js

```diff
@@ -1,11 +1,11 @@
 export const getUrlSmartTrader = () => {
-    // TODO: [app-link-refactor] - Remove backwards compatibility for `deriv.app`
-    if (
-        /^staging\.deriv\.app$/i.test(window.location.hostname) ||
-        /^staging-app\.deriv\.com$/i.test(window.location.hostname)
-    ) {
+    if (/^staging-app\.deriv\.com$/i.test(window.location.hostname)) {
```

Comment:
move the regexes to shared functions and share here and in config.js? (and url.js)

Reply:
Thanks for the suggestion. I've made the changes. :pray: